latest/docs-ref-conceptual/get-started-with-azure-cli.md

```diff
@@ -1,442 +1,88 @@
-resources in the "MyResourceGroup" resource group that we've used for all the samples in this Get Started tutorial.
-
-```azurecli-interactive
-az group delete -n MyResourceGroup
-```
-
-```Output
-Are you sure you want to perform this operation? (y/n): y
-```
+For more information, see the detailed documentation on [queries](query-azure-cli.md) and [output formats](format-output-azure-cli.md).
 
 ## Get samples
 
-To learn more about ways to use the Azure CLI, check out our most common scripts for
+To learn more about ways to use the Azure CLI, check out some samples demonstrating common things done with Azure services. Our most popular
+samples are for
 [Linux VMs](/azure/virtual-machines/virtual-machines-linux-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json),
-[Windows VMs](/azure/virtual-machines/virtual-machines-windows-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json), 
-[Web apps](/azure/app-service-web/app-service-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json),
-and [SQL Database](/azure/sql-database/sql-database-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json).
-
-## Read the API reference docs
-
-[API reference](/cli/azure)
-
-## Get help
-
-The Azure CLI has built-in help documentation, which matches our web documentation that you can run from the command line:
-
-```azurecli-interactive
-az [command-group [command]] -h
-```
+[Windows VMs](/azure/virtual-machines/virtual-machines-windows-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json),
+[Web apps](/azure/app-service-web/app-service-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json), and
+[SQL Database](/azure/sql-database/sql-database-cli-samples?toc=%2fcli%2fazure%2ftoc.json&bc=%2fcli%2fazure%2fbreadcrumb%2ftoc.json).
 
-For example, to see what commands and subgroups are available for VMs, use:
+## Tutorial
 
-```azurecli-interactive
-az vm -h
-```
-
-To get help with the command to create a VM, use:
-
-```azurecli-interactive
-az vm create -h
-```
+There is also a tutorial designed to help users get started with a simple CLI task. This tutorial offers an in-browser experience integrated
+with the Azure Cloud Shell to make it easy to run and understand.
 
-## Switch from Azure CLI 1.0
-
-If you already know how to use Azure CLI 1.0 (azure.js),
-you'll notice places where the commands aren't quite the same.
-Sometimes the commands to perform a task are significantly different.
-To help you make the switch from Azure CLI 1.0 to Azure CLI 2.0,
-we've started this [command mapping](https://github.com/Azure/azure-cli/blob/master/doc/azure2az_commands.rst).
-
-## Send us your feedback
-
-```azurecli-interactive
-az feedback
-```
+Go ahead and get started with the [Deploy a web server in Azure](tutorial-azure-cli-server.yml) tutorial.
```

Comment:
Just in general-where does this updated get started article really fit in with the user onboarding experience? It's get started but the only action you really are taking in learning the tool is logging in-the rest feels more like a tour.

Perfectly OK to ask Jason Shaver where he wants this article to go.

Reply:
My personal experience with "onboarding" a CLI is generally using it to do a single, common task that it was designed for (such as package managers or compiler tools.) The Azure CLI is different because it covers such a huge breadth of services and every user is going to come to it wanting something different. The only 'common task' that is shared is creating resource groups, a topic which should have a section in here.

The information that would probably help most for determining the average onboarding experience would be:

* Telemetry from @JasonRShaver about the most commonly-run CLI commands
* Telemetry from services overall about the most commonly used services

Even then, "Getting Started" might just serve best as this very small introductory article which is a dumping ground that links out to other content.